### PR TITLE
Update mvnbuild.adoc

### PR DESCRIPTION
**Pull request by JunyiSun**

### CODE REVIEW

**Comment by JunyiSun** [comment]
Please take a look. Thank you! @yeekangc @evelinec

**Comment by sishida** [comment]
@JunyiSun: I would change the second file edit to:
Then, verify that the tests pass by using the Maven `verify` goal in the `start` directory:
or
Then, use the Maven `verify` goal in the `start` directory to verify that the tests pass:

**Comment by yeekangc** [comment]
The 1st edit looks fine.

The 2nd edit needs work.  It should be moved up to before stop-server or be made part of it.

"If the server is still running from the previous steps, from the `start` directory, stop it first using the Maven `liberty:stop-server` goal:"

Or, "If the server is still running from the previous steps, stop it first using the Maven `liberty:stop-server` goal from command line in the `start` directory:"

**Comment by yeekangc** [comment]
Alternative is we add to the top of the section something like "Run the following commands from the `start` directory."  Or, add it to the end of the section.

File: mvnverify.adoc [hunk 1]
```diff
@@ -15,7 +15,7 @@ If the server is still running from the previous steps, stop it first using the
 mvn liberty:stop-server
 ```
 
-Then, verify that the tests pass using the Maven `verify` goal:
+Then, verify that the tests pass using the Maven `verify` goal in the `start` directory:
```
Comment (yeekangc) [inline mvnverify.adoc]: Move this up.  Or, preface the whole section with the reminder to run under `start`.

**Comment by yeekangc** [comment]
The latest change looks fine.  I wonder if we should preface the whole section instead.  Defer to further feedback from @sishida and @evelinec.  Thank you, @JunyiSun.

File: mvnverify.adoc [hunk 2]
```diff
@@ -9,7 +9,7 @@
 ////
 === Running the tests
 
-If the server is still running from the previous steps, stop it first using the Maven `liberty:stop-server` goal:
+If the server is still running from the previous steps, stop it first using the Maven `liberty:stop-server` goal from command line in the `start` directory:
```
Comment (sishida) [inline mvnverify.adoc]: How about..."If the server is still running from the previous steps, stop it with the Maven `liberty:stop-server` goal from the command line in the `start` directory:"

Comment (sishida) [reply mvnverify.adoc]: I felt that "first" was already implied because it's the first step under this heading. :)

**Pull request merged**